views/register.js

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+import {
+  View,
+  Text,
+  TouchableOpacity,
+  StyleSheet,
+  Button,
+  TextInput,
+  Alert,
+} from "react-native";
+import { KeyboardAwareScrollView } from "react-native-keyboard-aware-scroll-view";
+
+import { createStackNavigator } from "@react-navigation/stack";
+import { NavigationContainer } from "@react-navigation/native";
+
+export default function LoginPage({ navigation }) {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [username, setUsername] = useState("");
```

Comment:
ig you flipped the role of username and email but doesn't impact the functionality at all. So you don't have to change unless you want to

Reply:
I will leave it as it is for the time being.